sim/firesim-lib/src/main/cc/bridges/tracerv.h

```diff
@@ -51,12 +51,25 @@ class tracerv_t : public streaming_bridge_driver_t {
   virtual bool terminate() { return false; }
   virtual int exit_code() { return 0; }
   virtual void finish() { flush(); };
+  static void serialize(const uint64_t *const OUTBUF,
```

Comment:
Can we coalesce `OUTBUF` and `bytes_received` into a `const std::vector<uint64_t>`?

Reply:
refactoring the body of serialize into using modern std::vector is outside the scope of this PR